### PR TITLE
LIVE-1692: video.ts test

### PR DESCRIPTION
**Pull request by marjisound**
## Why are you doing this?
subtask of [increase coverage ticket](https://theguardian.atlassian.net/browse/LIVE-1684)

### Test coverage
| | Before | After |
| - | ------ | ----- |
| Statements | 60.54%  | 61.67% (+0.24%)|
| Branches | 55.09% | 57.6% (0%)| 
| Functions | 56.74% | 57.8% (+0.87%) |
| Lines | 59.73% | 60.82% (+0.2%) |

### CODE REVIEW

**Review by joecowton1 — APPROVED** [review]

File: src/video.test.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+		};
+	});
+
+	test('returns video', () => {
+		const expected = some({
+			posterUrl: 'poster-url',
+			videoId: 'asset-id',
+			duration: undefined,
+			atomId: 'atomId2',
+			title: 'mediaTitle',
```
Comment (faresite) [inline src/video.test.ts:79]: It's a nit but I'd reference these values as common variables so they live in one place?
Something like top level:
```
// in the same scope as `let blockElement: BlockElement;`
const atomId = 'atomId2'

const expected = some({
	...
	atomId,
}
```
If that makes sense?

**Comment by faresite** [comment]
Nice one @marjisound 👍  just a small nit I've added.

File: src/video.test.ts (hunk 1 above)
Comment (marjisound) [reply src/video.test.ts:79]: Yes that makes sense. Will do the change and let you know :)

File: src/video.test.ts [hunk 2]
```diff
@@ -30,7 +35,7 @@ describe('parseVideo', () => {
 				},
 			],
 			contentAtomTypeData: {
-				atomId: 'atomId2',
+				atomId: atomId,
```
Comment (faresite) [inline src/video.test.ts:38]: If the object key has the same name as the value variable you can shorthand it like so:
```
contentAtomTypeData: {
	atomId,
	atomType: 'someAtomType',
}
```

**Review by faresite — APPROVED** [review]
Nice one @marjisound 👍

File: src/video.test.ts (hunk 2 above)
Comment (marjisound) [reply src/video.test.ts]: Thanks Fares for pointing it out

**Pull request merged**